build.sbt

```diff
@@ -1,16 +1,78 @@
+import scala.collection.immutable.Seq
+
 lazy val `chatops4s` = (project in file("."))
   .settings(commonSettings)
+  .settings(
+    publish / skip := true
+  )
   .aggregate(
+    `chatops4s-core`,
+    `chatops4s-slack`,
+    `chatops4s-examples`,
     `chatops4s-discord`,
     `chatops4s-discord-example`,
   )
+lazy val `chatops4s-core` = (project in file("modules/chatops4s-core"))
+  .settings(commonSettings)
+  .settings(
+    libraryDependencies ++= Seq(
+      "org.typelevel" %% "cats-effect" % "3.6.2",
+      "org.typelevel" %% "cats-core" % "2.13.0"
+    ),
+    Test / parallelExecution := false
+  )
+
+lazy val `chatops4s-slack` = (project in file("modules/chatops4s-slack"))
+  .settings(commonSettings)
+  .settings(
+    libraryDependencies ++= Seq(
+      "com.softwaremill.sttp.client4" %% "core" % "4.0.9",
+      "com.softwaremill.sttp.client4" %% "cats" % "4.0.9",
+      "com.softwaremill.sttp.client4" %% "circe" % "4.0.9",
+      "com.softwaremill.sttp.client4" %% "httpclient-backend" % "4.0.9",
+      "io.circe" %% "circe-core" % "0.14.14",
+      "io.circe" %% "circe-generic" % "0.14.14",
+      "io.circe" %% "circe-parser" % "0.14.14",
+      "com.github.pureconfig" %% "pureconfig-core" % "0.17.9",
+      "com.github.pureconfig" %% "pureconfig-cats-effect" % "0.17.9",
+      "ch.qos.logback" % "logback-classic" % "1.5.18",
+      "org.typelevel" %% "log4cats-slf4j" % "2.7.1",
```

Comment:
Same here. while log4cats and logback are ok for application, they are too much for a lib. We will trim this down to pure slf4j later on.